agent/lib_guzzle6.c

```diff
@@ -342,14 +342,62 @@ const zend_function_entry nr_guzzle6_requesthandler_functions[]
 
 /* }}} */
 
+static char* nr_guzzle_version(zval* app TSRMLS_DC) {
+  char* retval = NULL;
+  zval* version = NULL;
+  zend_class_entry* ce = NULL;
+
+  if (0 == nr_php_is_zval_valid_object(app)) {
+    nrl_verbosedebug(NRL_FRAMEWORK, "%s: Application object is invalid",
+                     __func__);
+    return NULL;
+  }
+
+  ce = Z_OBJCE_P(app);
+  if (NULL == ce) {
+    nrl_verbosedebug(NRL_FRAMEWORK, "%s: Application has NULL class entry",
+                     __func__);
+    return NULL;
+  }
+
+  if (NULL == nr_php_get_class_constant(ce, "VERSION")) {
```

Comment:
Seems like this could be cleaned up if you just did
```
  version = nr_php_get_class_constant(ce, "VERSION");

  if (NULL == version) {
    nrl_verbosedebug(NRL_FRAMEWORK, "%s: Application does not have VERSION",
                     __func__);
    return " ";
  }
```

Reply:
Fixed: [5b6bd95](https://github.com/newrelic/newrelic-php-agent/pull/775/commits/5b6bd95c7be35f15731e3321cd6328973bef5c6d), I didn't change the return statement because when `NULL` is returned as the version, then `nr_php_packages_create()` will check for this and set the version to an empty string: https://github.com/newrelic/newrelic-php-agent/blob/65ca1de83400be2c7bc6bc0dbce28b1ff56ecd1d/axiom/nr_php_packages.c#L42